x-pack/plugins/ml/public/application/services/ml_api_service/results.ts

```diff
@@ -24,8 +24,9 @@ export const resultsApiProvider = (httpService: HttpService) => ({
     latestMs: number,
     dateFormatTz: string,
     maxRecords: number,
-    maxExamples: number,
-    influencersFilterQuery: any
+    maxExamples?: number,
+    influencersFilterQuery?: any,
+    actualPlotFunction?: string | undefined
```

Comment:
Maybe this should be renamed to `functionDescription` to match the name of the property in the records?

Reply:
Updated here https://github.com/elastic/kibana/pull/81662/commits/cb2429e192079f4e9a1cfc0e401a2a9544d19f23